screens/SplashScreen.js

```diff
@@ -1,10 +1,14 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { View, Image, TouchableOpacity, Dimensions, StyleSheet } from 'react-native';
 
 import Background from '../components/background';
 import NurseEllieLogo from '../assets/images/nurse-ellie-logo.svg';
-
+import { firebase } from "../components/Firebase/config";
 const SplashScreen = ({navigation}) => {
+   
+    firebase.auth().signOut();
```

Comment:
```suggestion
    useEffect(()=> {
    firebase.auth().signOut();
    }, []);
```
Put it in a useeffect